content/docs/conditional-rendering.md

```diff
@@ -1,243 +1,243 @@
 ---
 id: conditional-rendering
-title: Conditional Rendering
+title: Υπό όρους Rendering
```

Comment:
Δεν είμαι σίγουρος για το υπο όρους.
Συνήθως στα βιβλία conditionals είναι οι συνθήκες.
`Rendering με συνθήκες`?
`Υπό όρους rendering` μπορεί να συμαίνει ότι θα γίνει render ανάλογα με κάποιες συνήκες. Δλδ υπάρχει περίπτωση να μη γίνει render? Not sure... What do you think?

Reply:
@gatzjames Αυτό ακριβώς σκέφτηκα και εγώ. Μήπως να χρησιμοποιήσουμε τη έκφραση "Υποθετικό Rendering"?

Reply:
@gatzjames Καλύτερα πιστεύω `Υπό συνθήκη`.